www/kanban/jkanban.js

```diff
@@ -144,15 +145,18 @@
 
                         //Init Drag Item
                         self.drake = self.dragula(self.boardContainer, {
-                                moves: function (el, source, handle, sibling) {
-                                    if (self.options.readOnly) { return false; }
-                                    return handle.classList.contains('kanban-item');
-                                },
-                                accepts: function (el, target, source, sibling) {
-                                    if (self.options.readOnly) { return false; }
-                                    return true;
-                                },
-                                revertOnSpill: true
+                            moves: function (el, source, handle, sibling) {
```

Comment:
this looks like mostly changes to whitespace. Is it possible to minimize the changes to functional changes.

Reply:
I will try to convince my IDE about that